vignettes/HouseHoldCoanalysis.Rmd

```diff
@@ -0,0 +1,248 @@
+# Output variables
+
+## alignedTimesieres
+
+In the GGIR output folder `.../results/part6HouseholdCoAnalysis/alignedTimeseries`
+you will find csv files with the time series per household. 
+
+The data dictionary below shows the column names you would get for a household with
+two members: X and Y. Most of the columns are copied from the time series output files, which
+are documented [here](https://cran.r-project.org/web/packages/GGIR/vignettes/GGIR.html#571_Time_series_output_files).
+Therefore, those column are not documented here.
+
+Variable name | Description
+---- | ----------
+timenum | Timestamp for tSeconds since since 1-1-1970
+HID | Household ID
+ACC.X | Average magnitude of acceleration for household member X
+SleepPeriodTime.X | -
+invalidepoch.X | -
+guider.X | -
+window.X | -
+class_id.X | -
+lightmean.X | Mean light sensor value for member X
+lightpeak.X | Peak light sensor value for member X
+temperaturemean.X | Mean temperature value for member X
+onset.X | 1 to indicate sleep onset time for member X
+wakeup.X | 1 to indicate waking up for member X
+validepoch.X | 1 if epoch was valid
+ACC.Y | Average magnitude of acceleration for household member X
+SleepPeriodTime.Y | -
+invalidepoch.Y | -
+guider.Y | -
+window.Y | -
+class_id.Y | -
+lightmean.Y | See lightmean.X but now for member Y
+lightpeak.Y | See lightpeak.X but now for member Y
+temperaturemean.Y | -
+onset.Y | -
+wakeup.Y | -
+validepoch.Y |	 -
+time_POSIX | Timestamp in character format
+N_valid_hhmembers | Number of valid household member pairs
+validpair_X_Y | TRUE or FALSE to indicate whether the pair had valid data for this timestamp
+
+
+## Pairwise summary report
+
+As stored inside `pairwise_summary_all_housholds.csv`
+
+Variable name | Description
+---- | ----------
+HID	| Household ID
+Npairs	| Number of unique pairs in a household
+PID	| Pair ID
+MID1	| Household member ID for the first person in the pair
+MID2	| Household member ID for the second person in the pair
+event	| Number to count the number of events, for now these are days with valid wake-up estimates in both individuals
+time1	| Time person 1 wakes up
+time2	| Time person 2 wakes up
+date1	| Date person 1 wakes up
+date2	| Date person 2 wakes up
+index_wake1	| Index in the time series when person 1 woke up (probably not relevant for end-user)
+index_wake2	| Index in the time series when person 2 woke up (probably not relevant for end-user)
+firstawake | Who of the two individuals woke up first? If equal then value will be "equal"
+acc_wake1_before_wake2 | Mean acceleration of the person who wakes up first during the four minutes before the second person wakes up
+acc_wake2_before_wake2 | Mean activity of second person to wake up before they woke up
+lux_wake1_before_wake2 | Peak LUX of the person who wakes up first during the four minutes before the second person wakes up
+lux_wake2_before_wake2 | Peak LUX of second person to wake up before they woke up
+deltatime	| Difference in wake up time
```

Comment:
I think variable names here are lacking the unit of measurement. Is this minutes? seconds? Also for acceleration outcomes.

Reply:
Agree, I have now updated these.

For the LUX variables I am skipping this. The unit of light intensity is LUX, but I have tried to avoid the term light as light can be misunderstood as light activity. So, everywhere in GGIR where we discuss light exposure we call it lux. Not ideal, but I am not sure how else to do it.